article/sections/_9_conclusion.tex

```diff
@@ -0,0 +1,42 @@
+\section{Conclusion}
+\label{sec:conclusion}
+
+Travel demand problems aim at forecasting the impact of proposed project and policies. These problems are causal in nature. 
+To our surprise, there has been very little mention of causal inference in the field of travel demand modeling.
+\citet{brathwaite_2018_causal} have documented this disconnect and presented an initial framework for addressing it.
+
+In this chapter, we build up on \citet{brathwaite_2018_causal} and highlight the importance of using causal graphs and 
+causal inference methods in transportation demand modeling efforts.
+
+We presented a numerical exercise aiming to show the importance of  the data generating process in the estimation 
+of effects of interest resulting from external interventions. 
```

Comment:
I think the conclusion is one place we can afford to go with the less precise but more common "treatment" effect phrase, e.g.
```suggestion
of treatment effects from external interventions. 
```